<launch>
    <!-- pkg: 패키지 이름, type: 실행시킬 파일, name: 노드 이름, output: 터미널창에 띄우기 -->
    <node pkg="morai_woowa" type="control_node" name="control_node" output="screen"></node>
    <node pkg="morai_woowa" type="obstacle_tracking" name="obstacle_node"></node>
    <node pkg="morai_woowa" type="dwa_class" name="dwa_node" output="screen"></node>
    <node pkg="morai_woowa" type="path_tracking" name="path_tracking_node" output="screen"></node>
</launch>
